vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -40,7 +44,7 @@ def save_inventory(type, parent, hash)
     found = parent.send(type)
     if found.nil?
       found = parent.send("build_#{type}", hash.except(:id))
-      parent.send("#{type}=", found)
+      parent.update_attributes!(type => found)
```

Comment:
This one has been having me scratch my head...I think the following is much clearer and is the same...

```ruby
if found.nil?
  found = parent.send("create_#{type}", hash.except(:id))
else
  found.update_attributes!(hash.except(:id, :type))
end
```

The only thing I'm not sure about is if the `create_` supports the STI `:type` attribute in `hash`.  I'm double checking that.